ecs-cli/modules/utils/compose/name.go

```diff
@@ -45,14 +45,9 @@ func GetServiceName(prefix, projectName string) string {
 	return fmt.Sprintf("%s%s", prefix, projectName)
 }
 
-// GetTaskDefinitionFamily returns the task definition family name
-func GetTaskDefinitionFamily(prefix, projectName string) string {
```

Comment:
What happens if someone is still using the old config for read.

Reply:
It still works; there was a bug like you thought, but I fixed it. (I didn't re-add this function because I think its extraneous, but I did re-add the functionality it provided).